CONTRIBUTORS.md

```diff
@@ -16,3 +16,4 @@ Community
 [@EthanEChristian]: https://github.com/EthanEChristian
 [@kulkarnipushkar]: https://github.com/kulkarnipushkar
 [@dgoeke]: https://github.com/dgoeke
+[@sparkofreason]: https://github.com/sparkofreason
```

Comment:
I think you need to add your name to the list above for it to render in GitHub. The list here at the bottom looks to be a store of handles to links that isn't actually in the rendered markdown except in that it is joined with the rendered sections to create links.  I didn't find an entry for this sort of thing in the [GitHub markdown links section](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) but that appears to be what's going on (and when I made the change locally Atom rendered it as expected).